### DISCUSSION TITLE
Errors following documentation examples.

### DISCUSSION THREAD

**Discussion by thefacemyer** [QuSpin/QuSpin#539] [Q&A] [answered]
I am getting error when I try going through the documentation examples. For example, when trying to go through the [quspin.basis.spin_basis_1d](http://weinbe58.github.io/QuSpin/generated/quspin.basis.spin_basis_1d.html) documentation, I get errors saying "undefined Op" and "undefined ent_entropy"... I just updated QuSpin in Anaconda, and it still give the errors. Can anyone help me with correcting this?

**Comment by JiahaoYao** [comment, marked as answer]
can u show the whole error messages?

**Reply by thefacemyer** [reply]
![Op name not define quspin](https://user-images.githubusercontent.com/83994885/170071495-87690bff-5683-4392-9ee7-e3200e875667.png)

**Reply by thefacemyer** [reply]
I'm just reading through the documentation and copying and pasting the examples to work through it.

**Reply by JiahaoYao** [reply]
would u like to replace Op by basis.Op?

**Reply by thefacemyer** [reply]
I feel pretty dumb if that's the problem. Is that the rule? Don't just copy and paste the example code?
